src/univariates.jl

```diff
@@ -302,16 +302,16 @@ macro _delegate_statsfuns(D, fpre, psyms...)
     T = dt <: DiscreteUnivariateDistribution ? :Int : :Real
 
     # function names from StatsFuns
-    fpdf = symbol(string(fpre, "pdf"))
-    flogpdf = symbol(string(fpre, "logpdf"))
-    fcdf = symbol(string(fpre, "cdf"))
-    fccdf = symbol(string(fpre, "ccdf"))
-    flogcdf = symbol(string(fpre, "logcdf"))
-    flogccdf = symbol(string(fpre, "logccdf"))
-    finvcdf = symbol(string(fpre, "invcdf"))
-    finvccdf = symbol(string(fpre, "invccdf"))
-    finvlogcdf = symbol(string(fpre, "invlogcdf"))
-    finvlogccdf = symbol(string(fpre, "invlogccdf"))
+    fpdf = @compat(Symbol(fpre, "pdf"))
+    flogpdf = @compat(Symbol(fpre, "logpdf"))
+    fcdf = @compat(Symbol(fpre, "cdf"))
+    fccdf = @compat(Symbol(fpre, "ccdf"))
+    flogcdf = @compat(Symbol(fpre, "logcdf"))
+    flogccdf = @compat(Symbol(fpre, "logccdf"))
+    finvcdf = @compat(Symbol(fpre, "invcdf"))
+    finvccdf = @compat(Symbol(fpre, "invccdf"))
+    finvlogcdf = @compat(Symbol(fpre, "invlogcdf"))
+    finvlogccdf = @compat(Symbol(fpre, "invlogccdf"))
```

Comment:
```jl
               _
   _       _ _(_)_     |  A fresh approach to technical computing
  (_)     | (_) (_)    |  Documentation: http://docs.julialang.org
   _ _   _| |_  __ _   |  Type "?help" for help.
  | | | | | | |/ _` |  |
  | | |_| | | | (_| |  |  Version 0.4.5-pre+7 (2016-03-17 05:23 UTC)
 _/ |\__'_|_|_|\__'_|  |  Commit a5d90d8* (61 days old release-0.4)
|__/                   |  x86_64-apple-darwin15.4.0

julia> Symbol("A", "B")
ERROR: MethodError: `convert` has no method matching convert(::Type{Symbol}, ::ASCIIString, ::ASCIIString)
This may have arisen from a call to the constructor Symbol(...),
since type constructors fall back to convert methods.
Closest candidates are:
  convert(::Type{Symbol}, ::AbstractString)
  call{T}(::Type{T}, ::Any)
  convert{T}(::Type{T}, ::T)
 in call at essentials.jl:57
```